pythonforandroid/recipes/uvloop/__init__.py

```diff
@@ -0,0 +1,16 @@
+from pythonforandroid.recipe import PyProjectRecipe
+
+
+class UvloopRecipe(PyProjectRecipe):
+    version = 'v0.19.0'
+    url = 'git+https://github.com/MagicStack/uvloop'
```

Comment:
I can't remember if it will automatically checkout the tag this way, but if you don't want to pull the full repo history you could try that
```suggestion
    url = 'https://github.com/MagicStack/uvloop/archive/refs/tags/{version}.tar.gz'
```

Reply:
Tags were implemented recently: https://github.com/kivy/python-for-android/blob/344fd34d83b3e98c961dcd53b7d39606e1037593/pythonforandroid/recipe.py#L237

And we can't use source distribution because:
https://github.com/kivy/python-for-android/blob/344fd34d83b3e98c961dcd53b7d39606e1037593/pythonforandroid/recipe.py#L482-L486

(they don't contain .git folder)

Reply:
Nice thanks for the insight